lib/tools/android-manifest.js

```diff
@@ -89,18 +89,6 @@ manifestMethods.packageAndLaunchActivityFromManifest = async function (localApk)
   }
 };
 
-manifestMethods.targetSdkVersionFromManifest = async function (localApk) {
```

Comment:
@mykola-mokhnach, Actually this method add to implement to grant permissions.

As I mentioned in this [PR](https://github.com/appium/appium-adb/pull/187), grant permission feature depends on API versions. 

In legacy developed applications do not have these properties respected.